docs/helm-chart/production-guide.rst

```diff
@@ -184,6 +184,44 @@ To use an external StatsD instance:
       statsd_host: ...
       statsd_port: ...
 
+Datadog
+^^^^^^^
+If you are using a Datadog agent in your environment, this will enable Airflow to export metrics to the Datadog agent.
+NOTE: The below is for version 2.x.x of Airflow.  In Airflow 1.x.x the swap all instances of ``metrics`` for ``scheduler``
+
+.. code-block:: yaml
+
+  statsd:
+    enabled: false
+  config:
+    metrics:
```

Comment:
```suggestion
    metrics: # or 'scheduler' for Airflow 1
```

This is a cleaner approach to handling the Airflow 2 vs 1 differences I think? Then we can remove the note above?

Reply:
I agree.